server/job_stream_test.go

```diff
@@ -611,25 +570,27 @@ func TestRunStream_OrchestratorFailover(t *testing.T) {
 
 	//kick the second Orchestrator
 	go func() {
+		time.Sleep(200 * time.Millisecond)
```

Comment:
I recommend using [synctest](https://pkg.go.dev/testing/synctest) for all timing dependent tests, it makes things run [*much* faster](https://github.com/livepeer/go-livepeer/pull/3739) and we don't really want to make the test suite slower than it already is. It'll also surface some types of concurrency issues as long as the tests are run with`-race` (not sure if they already are, but if not, also recommended)

Reply:
Update: got all tests in `job_stream_test.go` to work with `synctest`.  Ran all locally with `count=10` successfully with `go test -timeout 30s -run  _BYOC$ github.com/livepeer/go-livepeer/server -count=10`